lib/knapsack_pro/runners/queue/rspec_runner.rb

```diff
@@ -119,6 +119,27 @@ def log_fail_fast_limit_met
 
         def post_trap_signals
           RSpec.world.wants_to_quit = true
+
+          print_current_rspec_batch_command
+        end
+
+        def post_log_threads(threads)
+          threads.each do |thread|
+            next unless thread.backtrace
+
+            spec_file_lines = thread.backtrace.select { |line| line.include?('_spec.rb') }
+            next if spec_file_lines.empty?
+
+            puts
+            if thread == Thread.main
+              puts "Hanging specs in the main thread:"
+            else
+              puts "Non-main thread inspect: #{thread.inspect}"
+              puts "Hanging specs in non-main thread:"
```

Comment:
plea

Hanging means that the code is actually stuck. But here it's about what's running when the method is executed. What about "Running specs"?